group_name: "cartesian_arm"
utopia_tolerance: 0.05
extend: false
max_distance: 0.1
use_kdtree: true
checker_resolution: 0.01
parallel_checker_n_threads: 4
path_file_name: "/config/path.yaml"
tree_file_name: "/config/tree.yaml"
